### PR TITLE
Resolves prettier and eslint issues

### PR DESCRIPTION
**Pull request by flashblaze**
Fixes #4

### CODE REVIEW

**Comment by flashblaze** [comment]
Any updates?

File: .eslintrc.json [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    }
-}
\ No newline at end of file
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true
+  },
+  "extends": ["prettier"],
+  "globals": {
```
Comment (arpitbatra123) [inline .eslintrc.json]: Can we get rid of these globals? I think these were added by eslint-config-google!

File: .eslintrc.json [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "max-len": ["error", { "code": 120 }],
+    "comma-dangle": "off"
```
Comment (arpitbatra123) [inline .eslintrc.json]: Might no longer need this rule since prettier will remove all trailing commas anyway!

**Review by arpitbatra123 — CHANGES_REQUESTED** [review]

**Comment by arpitbatra123** [comment]
@FlashBlaze Left some comments!

**Pull request closed without merging**